src/draw/__tests__/drawPolygonOnIjs.test.ts

```diff
@@ -129,7 +130,7 @@ describe('drawPolygon on IJS', () => {
     expect(result).not.toBe(image);
   });
   // the two following tests fail because there is a bug with filled polygons
```

Comment:
this comment can be removed now

Reply:
I had already done it in the latest commits.

Why is the comment still there when I click on the code snipped above, but not there when I go under the "Files changed" tab of the PR?

Reply:
Not sure, maybe I was looking at an outdated diff.